<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsdControllerWidget</class>
 <widget class="QWidget" name="UsdControllerWidget">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="StageRefLabel">
     <property name="text">
      <string>USD Stage:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1">
     <item>
      <widget class="QPushButton" name="PickStageButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>1</width>
         <height>30</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Select the USD stage node that contains the controller's source object.</string>
       </property>
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="ClearButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Clears the selected USD Stage node.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="ObjectPathLabel">
     <property name="toolTip">
      <string>The path of USD object used as source.</string>
     </property>
     <property name="text">
      <string>Object Path:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3">
     <item>
      <widget class="QLineEdit" name="ObjectPath"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
